webknossos/webknossos/administration/task.py

```diff
@@ -102,40 +88,31 @@ def create_from_annotations(
             len(base_annotations) > 0
         ), "Must supply at least one base annotation to create tasks"
 
-        client = _get_generated_client(enforce_auth=True)
-        url = f"{client.base_url}/api/tasks/createFromFiles"
-        task_parameters = {
-            "taskTypeId": task_type_id,
-            "neededExperience": {
-                "domain": needed_experience_domain,
-                "value": needed_experience_value,
-            },
-            "pendingInstances": instances,
-            "projectName": project_name,
-            "scriptId": script_id,
-            "boundingBox": bounding_box.to_wkw_dict()
```

Comment:
I think it is still used in the datasource-properties de/serialization code (also on disk, not only for the api client) I don’t want to touch that here.

A to_api_bounding_box method might make sense, but I kind of did not like to add such a dependency in BoundingBox. Do you think it would be better to include it?